SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
```

Comment:
Please don't put this in the root directory
1. we'd like to _reduce_ what we have in the root, not increase it (some are expected to move, just hasn't happened yet)
2. this is related to the test packages, specific to the `\test` tree

`\test\tools` seems appropos